test/cog/rule_ingestion_test.exs

```diff
@@ -77,7 +76,8 @@ defmodule RuleIngestionTest do
     permission("s3:delete")
     inserted_rule = rule("when command is cog:s3 with option[op] == 'delete' must have s3:delete")
 
-    %Cog.Models.Rule{parse_tree: retrieved_tree} = Queries.Command.rules_for(command) |> Cog.Repo.one!
+    command = Repo.preload(command, :rules)
```

Comment:
If you're only going to be working with rules after this, using an `assoc` query might be more intention revealing.

```
rules = Repo.all(assoc(command, :rules))
```